Frontend/src/pages/crowdactions/Detail/SocialMediaSharing.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { EmailIcon, EmailShareButton, FacebookIcon, FacebookShareButton, FacebookShareCount, LinkedinIcon, LinkedinShareButton, TumblrIcon, TumblrShareButton, TumblrShareCount, TwitterIcon, TwitterShareButton, WhatsappIcon, WhatsappShareButton } from 'react-share';
+import { ICrowdaction } from '../../../api/types';
+
+import styles from './SocialMediaSharing.module.scss';
+
+const SocialMediaSharing = ({ crowdaction }: any) => {
+
+  const action = crowdaction as ICrowdaction;  
+  const shareUrl = window.location.href;
+
+  return (
+    <div className={styles.shareButtons}>
```

Comment:
These components call the facebook/... API, there are likely privacy implications. A number of things are needed:

* Use the useConsent hook to check for social permissions before adding these components
* Check the privacy policy to check if we're allowed to do this (probably yes, but good to check)
* Might be good to ask ourselves if we really want the counters, might be easier to disable them. Something to discuss at the call?

Reply:
Counters are disabled, since they are not in the design either. Components are not calling any outside APIs.